membership-attribute-service/app/loghandling/ZuoraRequestCounter.scala

```diff
@@ -0,0 +1,15 @@
+package loghandling
+
+import java.util.concurrent.atomic.AtomicInteger
+
+import com.typesafe.scalalogging.LazyLogging
+
+object ZuoraRequestCounter extends LazyLogging{
+  private val counter = new AtomicInteger()
+
+  def increment: Int = counter.incrementAndGet()
```

Comment:
for neatness, maybe just get rid of these convenience methods, and move the `withZuoraRequestCounter` into this class instead.